### PR TITLE
106 loggedin

### LINKED ISSUE #106: Redirect User if they attempt to access "index" when they are already logged in
Should user see landing page or be redirected

### PR DESCRIPTION
**Pull request by Rebecca-CYLi**
Fixes #106 

Wait a min, will need to change some file variables back as I was testing them on my pc. ok, review ready for this PR

### CODE REVIEW

File: flask/app/rps.py [hunk 1]
```diff
@@ -16,3 +16,8 @@ def play():
 @rps.route("/stats")
 def stats():
     return render_template("stats.html")
+
+
+@rps.route("/mainpage")
```
Comment (calvh) [inline flask/app/rps.py:21]: what is this page for? I thought we should just send the user to the "play" page

Comment (Rebecca-CYLi) [reply flask/app/rps.py:21]: just a new index page if they accidentally clicked out of game and no where to enter again

Comment (calvh) [reply flask/app/rps.py:21]: what do you mean by "clicked out of game"? how does that happen?

Comment (Rebecca-CYLi) [reply flask/app/rps.py:21]: right now, if the player were to click on my stats, they wont be able to get back to the index page much less get back into the game.

Comment (calvh) [reply flask/app/rps.py:21]: I fixed this by adding a link to play in the navbar in @HamdiaA's PR (which is now merged)

On the index page, there are only login/register/play buttons so unless you want to write code to check if a player is logged in and hide those things I suggest simply redirecting the player to the play page if they try to access the main page.

**Comment by calvh** [comment]
I am closing this PR because the actual needed changes that have been committed so far are very minor and there are other changes, such as checking if the user is logged in when accessing the main page and redirecting them to the play page, are not present. In addition, there seem to be some unrelated commits like updating docker compose, updating localhost.conf files. Please create a new issue and a new PR.

**Pull request closed without merging**